core/discord_bot.py

```diff
@@ -169,42 +171,33 @@ async def _process_invites(self):
         print("Discord > Invite processor has been stopped.")
 
     async def _send_message(self, *args, **kwargs) -> Union[discord.Message, discord.WebhookMessage, None]:
+        self.name = DiscordConfig.serverName if DiscordConfig.serverName is not "" else "Bridge Bot"
         kwargs["allowed_mentions"] = discord.AllowedMentions.none()
-        if kwargs.pop("officer", False):
-            if self.officer_webhook:
-                kwargs["wait"] = True
-                try:
-                    return await self.officer_webhook.send(*args, **kwargs)
-                except Exception as e:
-                    print(f"Discord > Failed to send message to officer webhook: {e}")
-                    await self.send_debug_message(traceback.format_exc())
-            else:
-                channel = self.get_channel(DiscordConfig.officerChannel)
-                if channel is None:
-                    return
-                try:
-                    return await channel.send(*args, **kwargs)
-                except Exception as e:
-                    print(f"Discord > Failed to send message to officer channel {channel}: {e}")
-                    await self.send_debug_message(traceback.format_exc())
+        if not args and 'content' not in kwargs and 'embed' not in kwargs:
+            print("Discord > Warning: Attempted to send an empty message")
+            return None
+
+        is_officer = kwargs.pop("officer", False)
+        webhook = self.officer_webhook if is_officer else self.webhook
+        channel_id = DiscordConfig.officerChannel if is_officer else DiscordConfig.channel
```

Comment:
move this down into the `else` block as it isn't needed outside of it

Reply:
done